### PR TITLE
OCPBUGS#18168: Updated a note in creating volume snapshots proc

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.12+

Issue:
[OCPBUGS-18168](https://issues.redhat.com/browse/OCPBUGS-18168)

Link to docs preview:
[Preview](https://73359--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-snapshots#persistent-storage-csi-snapshots-create_persistent-storage-csi-snapshots)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
@brandisher 
Could you please review this PR?
Thanks.

**Comment by sr1kar99** [comment]
@radeore 
Can you please review this PR?
Thanks.

File: modules/persistent-storage-csi-snapshots-create.adoc [hunk 1]
```diff
@@ -15,9 +15,9 @@ When you create a `VolumeSnapshot` object, {product-title} creates a volume snap
 * A storage class to provision the storage back end.
 * No pods are using the persistent volume claim (PVC) that you want to take a snapshot of.
 +
-[NOTE]
+[IMPORTANT]
 ====
-Do not create a volume snapshot of a PVC if a pod is using it. Doing so might cause data corruption because the PVC is not quiesced (paused). Be sure to first tear down a running pod to ensure consistent snapshots.
+Creating a volume snapshot of a PVC that is in use by a pod can cause unwritten (cached) data to be excluded from the snapshot. To ensure that all data is written to the disk, delete the pod that is using the PVC before creating the snapshot.
```
Comment (kcarmichael08) [inline modules/persistent-storage-csi-snapshots-create.adoc:20]:
```suggestion
Creating a volume snapshot of a PVC that is in use by a pod can cause unwritten cached data to be excluded from the snapshot. To ensure that all data is written to the disk, delete the pod that is using the PVC before creating the snapshot.
```
I know IBM Style recommends avoiding parens and I don't think you need one here, but it's your call. https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses

**Review by kcarmichael08 — COMMENTED** [review]
This is a lot less clunky and awkward than the existing text. I have one suggestion but your call.

File: modules/persistent-storage-csi-snapshots-create.adoc (hunk 1 above)
Comment (sr1kar99) [reply modules/persistent-storage-csi-snapshots-create.adoc:20]: @brandisher 
Is it technically ok to have "unwritten cached" data?

Comment (brandisher) [reply modules/persistent-storage-csi-snapshots-create.adoc:20]: We can say `unwritten or cached` which is technically correct and drops the parens.

Comment (sr1kar99) [reply modules/persistent-storage-csi-snapshots-create.adoc]: Based on the discussion with @brandisher , updated it to "unwritten data and cached data" as both are different.

File: modules/persistent-storage-csi-snapshots-create.adoc [hunk 2]
```diff
@@ -15,9 +15,9 @@ When you create a `VolumeSnapshot` object, {product-title} creates a volume snap
 * A storage class to provision the storage back end.
 * No pods are using the persistent volume claim (PVC) that you want to take a snapshot of.
 +
-[NOTE]
+[IMPORTANT]
```
Comment (bburt-rh) [inline modules/persistent-storage-csi-snapshots-create.adoc:18]:
```suggestion
[WARNING]
```
- Per [our SSG guidelines](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions), I think this needs to be a WARNING admonition, which should be used for "Information about potential system damage, data loss, or a support-related issue if the user disregards this admonition." 

Can you please change this, and then ping me again for a merge? Thanks!

Comment (sr1kar99) [reply modules/persistent-storage-csi-snapshots-create.adoc]: Updated as per the suggestion.

**Pull request merged**